wp-security-audit-log/custom-alerts.php

```diff
@@ -272,6 +273,22 @@
 				'read',
 			),
 
+			array(
+				5717,
+				WSAL_MEDIUM,
+				esc_html__( 'An entry was created edited', 'wsal-gravity-forms' ),
```

Comment:
Created or edited?

Should this be actually like the following?
```suggestion
				esc_html__( 'An entry was edited', 'wsal-gravity-forms' ),
```